.github/workflows/docs.yml

```
@@ -0,0 +1,25 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+      - 'release-'
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
```

```suggestion
      - uses: julia-actions/setup-julia@v2
```

I think v2 is available since last year. Might as well use it.